DiscordManimator.py

```diff
@@ -104,29 +104,52 @@ async def manimate(ctx, *, arg):
                 container_stderr = dockerclient.containers.run(
                     image="manimcommunity/manim:stable",
                     volumes={tmpdirname: {'bind': '/manim/', 'mode': 'rw'}},
-                    command=f"timeout 120 manim /manim/script.py -qm -o scriptoutput {cli_flags}",
+                    command=f"timeout 120 manim /manim/script.py -qm --disable_caching -o scriptoutput {cli_flags}",
                     user=os.getuid(),
                     stderr=True,
                     stdout=False,
                     remove=True
                 )
                 if container_stderr:
-                    await ctx.reply("Something went wrong, here is "
-                                    "what Manim reports:\n"
-                                    f"```\n{container_stderr.decode('utf-8')}\n```")
+                    if len(container_stderr.decode('utf-8')) <= 1200:
+                        await ctx.reply("Something went wrong, here is "
+                                        "what Manim reports:\n"
+                                        f"```\n{container_stderr.decode('utf-8')}\n```")
```

Comment:
```suggestion
                        await ctx.reply(
                            "Something went wrong, here is "
                            "what Manim reports:\n"
                            f"```\n{container_stderr.decode('utf-8')}\n```"
                        )
```